z-schema/index.d.ts

```diff
@@ -1,12 +1,9 @@
 // Type definitions for z-schema v3.16.0
 // Project: https://github.com/zaggino/z-schema
-// Definitions by: Adam Meadows <https://github.com/job13er>
+// Definitions by: pgonzal <https://github.com/pgonzal>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export = ZSchema.Validator;
-export as namespace ZSchema;
-
-declare namespace ZSchema {
+declare module ZSchema {
```

Comment:
do not use `module` use `namespace`; they both mean the something, except that `namespace` is the preferred keyword now.